doc/releases/changelog-dev.md

```diff
@@ -4,29 +4,144 @@
+  [(#3900)](https://github.com/PennyLaneAI/pennylane/pull/3900)
+  [(#4000)](https://github.com/PennyLaneAI/pennylane/pull/4000)
+  [(#4004)](https://github.com/PennyLaneAI/pennylane/pull/4004)
+  TODO - check if performance PRs are merged.
+
+  This method can be activated by setting `diff_method` to
+  [`qml.gradient.stoch_pulse_grad`](https://docs.pennylane.ai/en/stable/code/api/pennylane.gradients.stoch_pulse_grad.html):
+
+  ```pycon
+  >>> dev = qml.device("default.qubit.jax", wires=2)
+  >>> sin = lambda p, t: jax.numpy.sin(p * t)
+  >>> ZZ = qml.PauliZ(0) @ qml.PauliZ(1)
+  >>> H = 0.5 * qml.PauliX(0) + qml.pulse.constant * ZZ + sin * qml.PauliX(1)
+  >>> @qml.qnode(dev, interface="jax", diff_method=qml.gradients.stoch_pulse_grad)
+  >>> def ansatz(params):
+  ...     qml.evolve(H)(params, (0.2, 1.))
+  ...     return qml.expval(qml.PauliY(1))
+  >>> params = [jax.numpy.array(0.4), jax.numpy.array(1.3)]
+  >>> jax.grad(ansatz)(params)
+  ```
+
+<h4>Quantum singular value transform 🐛➡️🦋</h4>
+
+* PennyLane now supports the
+  [quantum singular value transformation](https://arxiv.org/abs/1806.01838) (QSVT), which describes
+  how a quantum circuit can be constructed to apply a polynomial transformation to the singular
+  values of an input matrix.
+  [(#3756)](https://github.com/PennyLaneAI/pennylane/pull/3756)
+  [(#3757)](https://github.com/PennyLaneAI/pennylane/pull/3757)
+  [(#3758)](https://github.com/PennyLaneAI/pennylane/pull/3758)
+  [(#3905)](https://github.com/PennyLaneAI/pennylane/pull/3905)
+  [(#3909)](https://github.com/PennyLaneAI/pennylane/pull/3909)
+  [(#3926)](https://github.com/PennyLaneAI/pennylane/pull/3926)
+
+  Consider a matrix `A` along with a vector `angles` that describes the target polynomial
+  transformation. The [`qml.qsvt`](https://docs.pennylane.ai/en/stable/code/api/pennylane.qsvt.html)
+  function creates a corresponding circuit:
+
+  ```python
+  dev = qml.device("default.qubit", wires=2)
+
+  A = np.array([[0.1, 0.2], [0.3, 0.4]])
+  angles = np.array([0.1, 0.2, 0.3])
+
+  @qml.qnode(dev)
+  def example_circuit(A):
+      qml.qsvt(A, angles, wires=[0, 1])
+      return qml.expval(qml.PauliZ(wires=0))
+  ```
+  
+  This circuit is composed of
+  [`qml.BlockEncode`](https://docs.pennylane.ai/en/stable/code/api/pennylane.BlockEncode.html) and
+  [`qml.PCPhase`](https://docs.pennylane.ai/en/stable/code/api/pennylane.PCPhase.html) operations.
+
+  ```pycon
+  >>> qml.draw(example_circuit, expansion_strategy="device")(A)  # TODO
+  0: ─╭∏_ϕ─╭BlockEncode(M0)─╭∏_ϕ─╭BlockEncode(M0)†─╭∏_ϕ─┤  
+  1: ─╰∏_ϕ─╰BlockEncode(M0)─╰∏_ϕ─╰BlockEncode(M0)†─╰∏_ϕ─┤
+  ```
+
+  The [`qml.qsvt`](https://docs.pennylane.ai/en/stable/code/api/pennylane.qsvt.html) function
+  creates a circuit that is targeted at simulators due to the use of matrix-based operations.
+  Advanced users are able to use the
+  [`qml.QSVT`](https://docs.pennylane.ai/en/stable/code/api/pennylane.QSVT.html) template to perform
```

Comment:
Not sure that I love the nomenclature choice here (capitalized versus not), but, hey, c'est la vie!

Reply:
Hey Isaac, any other suggestions?